src/Service/ElasticaService.php

```diff
@@ -124,6 +127,99 @@ public function getAliasesFromIndex(string $indexName): array
         return $this->client->getIndex($indexName)->getAliases();
     }
 
+    /**
+     * @param array<mixed> $param
+     * @return CommonSearch
+     */
+    public function convertElasticsearchSearch(array $param): Search
+    {
+        @trigger_error("This function exists to simplified the migration to elastica, but should not be used on long term", E_USER_DEPRECATED);
```

Comment:
```suggestion
    public function fromLegacyElasticSearchParameters(array $param): Search
    {
```

Reply:
We could still keep the E_USER_DEPRECATED to track it in our logs. But it might polute our logs?

Reply:
A lot of admin projets use a `search` filter this filter is just a schortcut to the elasticseach search function. I can remove it from here as code upper in the stack should be also deprecated